bookwyrm/models/user.py

```diff
@@ -10,7 +10,7 @@
 from bookwyrm.models.status import Status
 from bookwyrm.settings import DOMAIN
 from bookwyrm.signatures import create_key_pair
-from .base_model import ActivityMapping, OrderedCollectionPageMixin
+from .base_model import ActivityMapping, OrderedCollectionPageMixin, PrivacyLevels
```

Comment:
I moved some things around in how models are structured, so you'll have to import PrivacyLevels from `.fields` instead of  `.base_model` -- sorry about that! (https://github.com/mouse-reeve/bookwyrm/commit/b6907f39e97529211f53ee175ed657f999261ecf)